File: pos-node/consumerNews.js
var soap = require('soap');
var url="http://localhost:8080/pos-news/NewsSOAPService?wsdl"   
news = {
        'arg0':{
          'id': '0',
          'title':'Java in veia!!! :)))) ',
          'auth':'afdafasfa',
          'body':'Java is greatttt!!!!!!!!!!'
        }
}
soap.WSDL.prototype.ignoredNamespaces = ['http://service.ifpb.edu.br/'];
soap.createClient(url, function(err, client) {
    client.persist(news, function(err,result,res){
         console.log('Retorno:',res);
        //  client.all(function(err,result,res){
        //       console.log(result.return);
        //  })
    })
    client.all(function(err,result,res){
         console.log(result.return);
         console.log('Retorno:',result);
    })
});

File: pos-python/consumerNews.py
from suds.client import Client
from suds.cache import NoCache
url="http://localhost:8080/pos-news/NewsSOAPService?wsdl"
client = Client(url, cache=NoCache())
# news = {
#     'id': '0',
#     'title':'JS in veia!!! :)))) ',
#     'auth':'Deitel',
#     'body':'Java is poor'
# }
class News:
    def __init__(self, title, auth, body ):
        self.title = title
        self.auth = auth
        self.body = body
        
news = News('Sample 2','too', 'too de novo')
#invocando os metodos
# client.service.persist(news)
print client.service.all()
#print client

